src/test/java/uk/gov/pay/adminusers/unit/service/ServiceResourceUpdateTest.java

```diff
@@ -503,4 +503,29 @@ public void shouldUpdateExperimentalFeaturesEnabled_toTrue() {
 
         assertThat(json.get("experimental_features_enabled"), is(true));
     }
+
+    @Test
+    public void shouldUpdateTakesPaymentsOverPhone_toTrue() {
+        ServiceEntity thisServiceEntity = ServiceEntityFixture
```

Comment:
Can we set `takes_payments_over_phone` to `false` explicitly when creating the service?

Reply:
oops. copy & paste error. now fixed.